dom/src/main/scala/org/scalajs/dom/Window.scala

```diff
@@ -445,4 +445,31 @@ class Window
     * number. You can get the number of pixels the document is scrolled horizontally from the scrollX property.
     */
   def scrollY: Double = js.native
+
+  /** Encodes a URI by replacing each instance of certain characters by one, two, three, or four escape sequences
+    * representing the UTF-8 encoding of the character (will only be four escape sequences for characters composed of
+    * two "surrogate" characters).
+    *
+    * @param uriComponent
+    *   A string, number, boolean, null, undefined, or any object. Before encoding, the uriComponent gets converted to
+    *   string.
+    *
+    * @return
+    *   A new string representing the provided uriComponent encoded as a URI component.
+    */
+  def encodeURIComponent(uriComponent: js.Any): String = js.native
```

Comment:
There's very few circumstances to use `js.Any`.

```suggestion
  def encodeURIComponent(uriComponent: Any): String = js.native
```